File: app/src/main/java/acr/browser/lightning/search/engine/BingSearch.java
package acr.browser.lightning.search.engine;

import acr.browser.lightning.constant.Constants;

/**
 * The Bing search engine.
 * <p>
 * See http://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/Bing_logo_%282013%29.svg/500px-Bing_logo_%282013%29.svg.png
 * for the icon.
 */
public class BingSearch extends BaseSearchEngine {

    public BingSearch() {
        super("file:///android_asset/bing.png", Constants.BING_SEARCH);
        // "http://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/Bing_logo_%282013%29.svg/500px-Bing_logo_%282013%29.svg.png";
    }

}

File: app/src/main/java/acr/browser/lightning/search/engine/BingSearch.kt
package acr.browser.lightning.search.engine

import acr.browser.lightning.R

/**
 * The Bing search engine.
 *
 * See http://upload.wikimedia.org/wikipedia/commons/thumb/b/b1/Bing_logo_%282013%29.svg/500px-Bing_logo_%282013%29.svg.png
 * for the icon.
 */
class BingSearch : BaseSearchEngine(
        "file:///android_asset/bing.png",
        "https://www.bing.com/search?q=",
        R.string.search_engine_bing
)
